lib/tutorials/getting-started.md

```diff
@@ -106,7 +106,10 @@ server.register({
     options: {
         reporters: [{
             reporter: require('good-console'),
-            args:[{ log: '*', response: '*' }]
+            events: {
+                request: '*',
```

Comment:
Why did you remove response and added request to log events? see https://github.com/hapijs/hapijs.com/issues/168